packages/web-components/src/components/footer/legal-nav.ts

```diff
@@ -76,15 +79,18 @@ class C4DLegalNav extends StableSelectorMixin(LitElement) {
     return this.size !== FOOTER_SIZE.MICRO
       ? html`
           <nav
+            part="legal-nav"
             class="${c4dPrefix}--legal-nav"
             aria-label="${ifDefined(navLabel)}">
-            <div class="${this._getTypeClass()}">
+            <div part="legal-nav-list" class="${this._getTypeClass()}">
               <ul>
                 <slot></slot>
               </ul>
               <slot name="locale"></slot>
             </div>
-            <div class="${c4dPrefix}--adjunct-links__container">
+            <div
+              part="adjunct-links-container"
+              class="${c4dPrefix}--adjunct-links__container">
               <ul>
```

Comment:
Missing a `part` attribute here. Recommend `adjunct-links-list`.

Also, review lines 100-111 below. Use the same `legal-nav*` part names as above lines 81-98. If you look at the Footer > Micro story, you can see how these are presently missing.